source/precalculus/source/01-EQ/05.ptx

```diff
@@ -680,13 +680,13 @@
   <task>
     <statement>
       <p>
-        When written in standard form, what are the values of <m>a</m>, <m>b</m>, and <m>c</m>?
+        When written in standard form, what are the values of <m>a</m>, <m>b</m>, and <m>c</m>? (Hint: rewrite the equation so that your <m>a</m>-value is positive.)
```

Comment:
There is a `<hint>` tag we can use for hints.  You can stick it between the `<statement>` and the `<answer>`

Reply:
Oh right! Added those in :)